magicbot/magicrobot.py

```diff
@@ -456,6 +456,31 @@ def _setup_vars(self, cname, component):
             setattr(component, n, injectable)
             self.logger.debug("-> %s as %s.%s", injectable, cname, n)
         
+        if hasattr(component, '__annotations__'):
```

Comment:
why not `for n, _type in getattr(component, '__annotations__', []).items()`? Save an indentation level.